coinflip/script.js

```diff
@@ -0,0 +1,30 @@
+let coin = document.querySelector(".coin");
+console.log(coin)
+let flipBtn = document.querySelector("#flip-button");
+let resetBtn = document.querySelector("#reset-button")
+
+ 
+
+let heads = 0;
+let tails = 0;
+
+flipBtn.addEventListener("click",() =>{
+    let i = Math.floor(Math.random() *2 );
```

Comment:
I don't think you re-assign the `i` variable later in this function. I would declare this with `const` always use `const` by default if possible.